src/style.css

```diff
@@ -1 +1,71 @@
 /* add styles here */
+html, body {
+  height: 100%;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+
+.stars__star {
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+/* .stars--0 .stars__star:not(:first-child) {
+  margin-left: 4px;
+} */
+
+.stars--1 .stars__star:first-child {
+  background-image: url(images/star-active.svg);
+}
+
+/* .stars--1 .stars__star:not(:first-child) {
+  margin-left: 4px;
+} */
+
+.stars--2 .stars__star:nth-child(-n+2) {
+  background-image: url(images/star-active.svg);
+}
+
+/* .stars--2 .stars__star:not(:first-child) {
+  margin-left: 4px;
+} */
+
+.stars--3 .stars__star:nth-child(-n+3) {
+  background-image: url(images/star-active.svg);
+}
+
+/* .stars--3 .stars__star:not(:first-child) {
+  margin-left: 4px;
+} */
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(images/star-active.svg);
+}
+
+/* .stars--4 .stars__star:not(:first-child) {
+  margin-left: 4px;
+} */
+
+.stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(images/star-active.svg);
+}
+
+/* .stars--5 .stars__star:not(:first-child) {
+  margin-left: 4px;
+} */
```

Comment:
remove comments and combine all the selectors with commas

Reply:
sure, thanks! my mistake